spec/models/project_spec.rb

```diff
@@ -301,4 +301,34 @@
       end
     end
   end
+
+  describe "completed scope" do
```

Comment:
Might be nice to wrap these in a `describe "Scopes"` which is what we have done in the other spec files